openadapt/visualize.py

```diff
@@ -168,6 +174,9 @@ def main():
         image = display_event(action_event)
         diff = display_event(action_event, diff=True)
         mask = action_event.screenshot.diff_mask
+        image = scrub.scrub_image(image)
```

Comment:
Please add a `config.SCRUB_ENABLED` parameter that disables scrubbing everywhere